src/xdp/trn_agent_xdp.c

```diff
@@ -62,6 +62,9 @@ static __inline int trn_select_transit_switch(struct transit_packet *pkt,
 	*s_port = SPORT_MIN + (inhash % SPORT_BASE);
 
 	if (*s_port < SPORT_MIN || *s_port > SPORT_MAX) {
+		bpf_debug("[Agent:] trn_select_transit_switch (BUG): s_port [%d] "
```

Comment:
We should keep "Agent:%ld.0x%x" to distinguish node, vpc. There should be enough args left. If not, can this be condensed?

Reply:
This is a copy/paste from L129. I'll address both places.

Reply:
Actually L81 as well.